FWCore/ParameterSet/python/Config.py

```diff
@@ -554,6 +558,41 @@ def extend(self,other,items=()):
                 newSeq._place(name,self)
         self.__dict__['_Process__InExtendCall'] = False
 
+    # FIXME define explicitly if the items should be referenced or copied
```

Comment:
It looks to me this can add and modify, but not delete an object:
if in "fragment" and object gets deleted, it is simply not there anymore.
How will "process" see this? You can not just delete in "process"
anything which is not in "fragment". Perhaps one needs in "fragment"
and explicit list of objects to be deleted, and run through it here?